### ISSUE TITLE
Minedojo seems to hang on `minedojo.make` on Windows

### ISSUE THREAD

**Issue by user670** [MineDojo/MineDojo#5]
```
C:\Users\User670>python
Python 3.10.0 (tags/v3.10.0:b494f59, Oct  4 2021, 19:00:18) [MSC v.1929 64 bit (AMD64)] on win32
Type "help", "copyright", "credits" or "license" for more information.
>>> import minedojo
[INFO:minedojo.tasks] Loaded 1572 Programmatic tasks, 1558 Creative tasks, and 1 special task: "Playthrough". Totally 3131 tasks loaded.
>>> env=minedojo.make(task_id="harvest_milk", image_size=(1,1))
```

Program stays here with no sign of moving, no logs printed, no window opened, no `>>>` prompting me to type the next line of code, for a really really long time.

Python.exe takes up 8GB of RAM, but not too much CPU.

Installation: installed by `pip install`. No conda virtual env. Java 8 separately installed from [here](https://www.oracle.com/cn/java/technologies/javase/javase8-archive-downloads.html), with `java -showversion` saying `java version "1.8.0_202"`.

Windows 10 x64 home edition. Running from a Command Prompt window in Windows Terminal.

(I do realize Minedojo is probably intended for a unix-like and probably doesn't support Windows, considering Windows is not mentioned in the docs...)

**edit**: After slapping many `print` statements in the minedojo code, the place where it hangs seems to be `sim/config_sim_spec.py` -> `SimSpec.__init__` where it calls `self._observation_space.seed(seed)`. I didn't go deeper than this yet, if I decided to go deeper and slap more `print`s, I'll update my findings here.

**edit 2**: I decided to slap some `print`s in gym. The performance hogger is a `self.np_random.choice` in `gym.spaces.Dict.seed`. Also somehow I observed it's not hanging, it's just very slow, taking at least a full minute to run this `choice` once, and apparently it needs to run multiple times throughout a `minedojo.make`.

**edit 3**: [numpy.random.RandomState.choice](https://numpy.org/doc/stable/reference/random/generated/numpy.random.RandomState.choice.html). The first parameter passed in was 2147483647. Wonder why. Also, if numpy expands this 2147483647 (`np.arange(2147483647)`) into an array of 2 billion `int`s, that'd perfectly account for the 8 GB of RAM it's taking.

**Comment by user670** [comment]
NOT SOLVED: Minedojo will not accept gym 0.24 as compatible. I tried to uninstall and re-install minedojo (because I want to revert the changes I made), and it UNINSTALLED gym 0.24 and installed gym 0.21.

**Comment by yunfanjiang** [comment]
Thanks for reporting this issue!

Please note that MineDojo was tested on Ubuntu 20.04 and MacOS intel chip. It may or may not work on other platforms. We welcome community contribution.

That said, we still asked one of our members who has access to Windows to test this issue. He found the exactly same reason as you did. Seeding a gym 0.21 space takes one minute on his machine.

We stick to gym 0.21 because it introduces lots of behavior changes since v0.22. We will remark this issue and update to be compatible with gym 0.22+ in the next version.
